cookbook/plugins/direct-database-actions/work-with-quantity/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Sample Plugin License
+
+Copyright 2021 Agilent Technologies Inc.
```

Comment:
Maybe we should check with Hannes -- does the year in this need to be updated? Or is the year here the year that the text was generated...